cache/remotecache/local/config.go

```diff
@@ -0,0 +1,67 @@
+package local
+
+import (
+	"strconv"
+
+	"github.com/moby/buildkit/util/compression"
+	digest "github.com/opencontainers/go-digest"
+	"github.com/pkg/errors"
+)
+
+const (
+	attrDigest           = "digest"
+	attrSrc              = "src"
+	attrDest             = "dest"
+	attrOCIMediatypes    = "oci-mediatypes"
+	contentStoreIDPrefix = "local:"
+)
+
+type Config struct {
+	Digest         digest.Digest
+	Store          string
+	OCIMediatypes  bool
+	ContentStoreID string
+	Compression    compression.Config
+}
+
+func getConfig(attrs map[string]string, export bool) (*Config, error) {
+	var store string
+	var dgst digest.Digest
+	if export {
```

Comment:
I think this would be neater if we avoid the conditional here, and let the caller check to see if src and dest are set - we could change the struct fields to be pointers to make it a bit easier for that.